### PR TITLE
docs: document allowed prefixes for branch name style action

### PR DESCRIPTION
**Pull request by jorgepiloto**
This pull-request documents in a `note` admonition the allowed branch name prefixes.

### CODE REVIEW

File: branch-name-style/action.yml [hunk 1]
```diff
@@ -5,6 +5,19 @@ description: >
...
+  .. note::
+
+      Branch names must start with one of the following prefixes:
+
+      - feat
+      - junk
+      - maint
+      - doc or docs
+      - no-ci
+      - testing
```
Comment (MaxJPRey) [inline branch-name-style/action.yml:17]: Can we support test?

I am used to it 😄 .

Comment (jorgepiloto) [reply branch-name-style/action.yml:17]: Sure!

File: branch-name-style/action.yml [hunk 2]
```diff
@@ -5,6 +5,19 @@ description: >
...
+      Branch names must start with one of the following prefixes:
+
+      - feat
+      - junk
+      - maint
+      - doc or docs
+      - no-ci
+      - testing or test
+      - release
+      - dependabot
```
Comment (Revathyvenugopal162) [inline branch-name-style/action.yml:19]: if possible, fix as well?

Comment (jorgepiloto) [reply branch-name-style/action.yml:19]: Good point, Revathy 🚀

**Comment by RobPasMue** [comment]
@jorgepiloto - accepted prefix is ``docs``, not ``doc``

**Comment by jorgepiloto** [comment]
> @jorgepiloto - accepted prefix is `docs`, not `doc`

Then, we need to change also this prefix in the actions.

**Review by Revathyvenugopal162 — APPROVED** [review]

**Pull request merged**